samples/csharp/getting-started/DatabaseIntegration/DatabaseIntegration/Program.cs

```diff
@@ -0,0 +1,102 @@
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms.Categorical;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+
+namespace DatabaseIntegration
+{
+    public class Program
+    {
+        private static string Download(string baseGitPath, string dataFile)
+        {
+            using (WebClient client = new WebClient())
+            {
+                client.DownloadFile(new Uri($"{baseGitPath}"), dataFile);
+            }
+
+            return dataFile;
+        }
+
+        public static string DownloadAdultDataset()
+            => Download("https://raw.githubusercontent.com/dotnet/machinelearning/244a8c2ac832657af282aa312d568211698790aa/test/data/adult.train", "adult.txt");
+
+        public static List<AdultCensus> GetAdultData()
+        {
+            string adultDataset = DownloadAdultDataset();
+            string dataLine;
+            var dataset = new List<AdultCensus>();
+            using (StreamReader reader = new StreamReader(adultDataset))
+            {
+                // read the header
+                dataLine = reader.ReadLine();
+                while ((dataLine = reader.ReadLine()) != null)
+                {
+                    if (string.IsNullOrEmpty(dataLine))
+                        continue;
+
+                    var data = dataLine.Split(',');
+                    dataset.Add(new AdultCensus()
+                    {
+                        Age = int.Parse(data[0]),
+                        Workclass = data[1],
+                        Education = data[3],
+                        //EducationNum = int.Parse(data[4]),
+                        MaritalStatus = data[5],
+                        Occupation = data[6],
+                        Relationship = data[7],
+                        Race = data[8],
+                        Sex = data[9],
+                        CapitalGain = data[10],
+                        CapitalLoss = data[11],
+                        HoursPerWeek = int.Parse(data[12]),
+                        NativeCountry = data[13],
+                        Label = int.Parse(data[14])
+                    });
+                }
+            }
+
+            return dataset;
+        }
+
+        public static void PopulateDatabase()
```

Comment:
NIT: You could use the word "seeding" instead of populating the database. It is a term familiar to many EF Core users.